### PR TITLE
Add clean_observers management commands

### PR DESCRIPTION
**Pull request by gregorjerse**
that cleans old subscriptions

### CODE REVIEW

File: resolwe/observers/management/commands/clean_observers.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ./manage.py clean_observers
+
+"""
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from resolwe.observers.models import Observer, Subscription
+
+# Delete subscriptions older than DELETE_OLDER_THAN days.
+DELETE_OLDER_THAN = 1
```
Comment (plojyon) [inline resolwe/observers/management/commands/clean_observers.py:18]: Specify threshold in hours to allow more granular control.

**Review by plojyon — CHANGES_REQUESTED** [review]

File: resolwe/observers/management/commands/clean_observers.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def handle(self, *args, **options):
+        """Delete old observer subscriptions."""
+        delete_older_than = options["older_than"] or DELETE_OLDER_THAN
+        # Delete subscriptions older than delete_older_than hours.
+        Subscription.objects.filter(
+            created__lt=timezone.now() - timezone.timedelta(hours=delete_older_than)
+        ).delete()
+        # Delete observers without subscriptions.
+        Observer.objects.filter(subscriptions__isnull=True).delete()
```
Comment (plojyon) [inline resolwe/observers/management/commands/clean_observers.py:43]: Add some logging so we know the cronjob ran?

**Review by plojyon — APPROVED** [review]
looks good

**Comment by gregorjerse** [comment]
Duplicate #1039

File: resolwe/observers/management/commands/clean_observers.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        )
+
+    def handle(self, *args, **options):
+        """Delete old observer subscriptions."""
+        delete_older_than = options["older_than"] or DELETE_OLDER_THAN
+        logger.info(
+            "Deleting observer subscriptions older than %d hours.", delete_older_than
+        )
+        # Delete subscriptions older than delete_older_than hours.
+        deleted_subscriptions = Subscription.objects.filter(
```
Comment (dblenkus) [inline resolwe/observers/management/commands/clean_observers.py:46]: It is more pythonic to write `deleted_subscriptions, _` instead of using `[0]`

**Pull request closed without merging**